governance/default_test.go

```diff
@@ -537,3 +537,26 @@ func TestCypressGenesisHash(t *testing.T) {
 		t.Errorf("Generated hash is not equal to Cypress's hash. Want %v, Have %v", cypressHash.String(), block.Hash().String())
 	}
 }
+
+func TestWriteGovernance_idxCache(t *testing.T) {
+	gov := getGovernance()
+
+	tstMap := copyMap(testGovernanceMap)
+
+	src := NewGovernanceSet()
+	delta := NewGovernanceSet()
+	src.Import(tstMap)
+
+	blockNum := []uint64{30, 30, 60, 60, 50}
+
+	for _, num := range blockNum {
+		if ret := gov.WriteGovernance(num, src, delta); ret != nil {
+			t.Errorf("Error in testing WriteGovernance, %v", ret)
+		}
+	}
+
+	// idxCache should have 0, 30 and 60
+	if len(gov.idxCache) != 3 || gov.idxCache[len(gov.idxCache)-1] != 60 {
```

Comment:
minor comment:
How about checking 0 and 30 also?

Reply:
@jeonghonah I also thought of it. But I think in this case, checking length of the cache is enough to cover that..